vlayer/examples/circuits/is_crypto_punk_owner/Prover.toml

```diff
@@ -0,0 +1,3 @@
+block_number = 14194126
+token_id = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 9]
```

Comment:
Maybe tokenId can be Field in cryptoPunks? Do they actually have that many ids that we need Bytes32?

Reply:
Changed